src/rgw/rgw_object_expirer_core.cc

```diff
@@ -147,6 +147,27 @@ int RGWObjExpStore::objexp_hint_list(const string& oid,
   return 0;
 }
 
+int RGWObjExpStore::cls_timeindex_trim_repeat(rgw_rados_ref ref,
```

Comment:
this doesn't appear to need anything in RGWObjExpStore, so it can be a static function instead:
`static int cls_timeindex_trim_repeat(...`
(and removed from rgw_object_expirer_core.h)